.github/workflows/ci.yml

```diff
@@ -45,7 +45,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [ "3.7", "3.8", "3.9", "3.10", "3.11", "3.12" ]
+        python-version:
+          - "3.7"
```

Comment:
IMHO you can remove `3.7` that is not supported anymore (it's been a year).

Once you add python 3.13, you could also drop python3.8 (that is official reaching EOL this October, like, in 7 days)

Reply:
Thanks for the suggestion. I think that's up to the maintainer. If the burden of keeping it around is not too big, they might wanna wait a while before dropping it.

Reply:
Indeed. No strong reason to remove its support.